### PR TITLE
Develop emission db

### PR DESCRIPTION
**Pull request by mefuru**

### CODE REVIEW

File: packages/emissions-dashboard/fixtures/saveFixturesToMongo.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const MongoClient = require('mongodb').MongoClient;
+const transactions = require('./transactionFixtures.json');
+require('@popcorn/utils/src/envLoader');
+
+const uri = `mongodb+srv://${process.env.MONGO_USERNAME}:${process.env.MONGO_PASSWORD}@cluster0.mcgwt.mongodb.net`;
```
Comment (anthonymartin) [inline packages/emissions-dashboard/fixtures/saveFixturesToMongo.js:5]: suggestion: replace entire uri with an environment variable so this isn't coupled to this mongo provider

File: packages/emissions-dashboard/netlify.toml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+[[plugins]]
+  package = 'netlify-plugin-contextual-env'
+  [plugins.inputs]
+    mode = 'prefix'
+
+[context.deploy-preview]
+
+  [context.deploy-preview.environment]
+    # this is a random private key needed for hardhat compile to work
+    PRIVATE_KEY = "31c18a7e6aaf93d1548ec9ae1b322c75400cb39627ae82bd1bb761ea79224a7d"
```
Comment (anthonymartin) [inline packages/emissions-dashboard/netlify.toml:17]: can you get rid of the stuff here that's irrelevant and from the other package?

File: packages/emissions-dashboard/netlify/functions/loadtransactions.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { MongoClient } from 'mongodb';
+
+exports.handler = async (event, context) => {
+  try {
+    const uri = 'mongodb://localhost:27017';
```
Comment (anthonymartin) [inline packages/emissions-dashboard/netlify/functions/loadtransactions.ts:5]: should this be localhost?

File: packages/emissions-dashboard/fixtures/saveFixturesToMongo.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const MongoClient = require('mongodb').MongoClient;
```
Comment (anthonymartin) [inline packages/emissions-dashboard/fixtures/saveFixturesToMongo.js:1]: we should only be using typescript files in our projects. is there a reason this one is js?

**Review by anthonymartin — CHANGES_REQUESTED** [review]

**Comment by anthonymartin** [comment]
is there a readme? it would be nice to describe syncing data, bootstrapping the project, etc.

**Comment by anthonymartin** [comment]
@mefuru we should also create a `.env.example` file that gets copied over to `.env` for env vars

**Comment by anthonymartin** [comment]
the `.env.example` file should be created within this packages directory to keep the env localized to this package

File: packages/emissions-dashboard/fixtures/saveFixturesToMongo.js (hunk 4 above)
Comment (mefuru) [reply packages/emissions-dashboard/fixtures/saveFixturesToMongo.js]: Just so I could test fixture grabs quickly. Updated to TS

File: packages/emissions-dashboard/netlify/functions/loadtransactions.ts (hunk 3 above)
Comment (mefuru) [reply packages/emissions-dashboard/netlify/functions/loadtransactions.ts]: Updated to db uri in .env

File: packages/emissions-dashboard/netlify.toml (hunk 2 above)
Comment (mefuru) [reply packages/emissions-dashboard/netlify.toml]: Done

File: packages/emissions-dashboard/fixtures/saveFixturesToMongo.js (hunk 1 above)
Comment (mefuru) [reply packages/emissions-dashboard/fixtures/saveFixturesToMongo.js]: Done

**Pull request closed without merging**